.circleci/config.yml

```diff
@@ -1,17 +1,18 @@
 version: 2.1
 orbs:
-  architect: giantswarm/architect@2.11.0
+  architect: giantswarm/architect@3.3.0
```

Comment:
This is fine, latest is 4.0.0 though (but no changes to the `push-to-app-catalog` job

Reply:
Done